images/loki/tests/02-use.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+echo "${IMAGE_NAME}"
+
+
+id=$(docker run -d --rm "${IMAGE_NAME}")
+sleep 10
+trap "docker kill ${id}" EXIT
+
+docker logs "${id}" 2>&1 | grep "Loki started"
```

Comment:
Is there a Helm chart for this, or a way we can actually test the log streaming?

Reply:
Look like: https://github.com/grafana/loki/tree/main/production/helm

Reply:
Actually that may be deprecated in favor of: https://github.com/grafana/helm-charts/tree/main/charts/loki-stack?

Reply:
I think they just moved the charts, trying to get that one running.